vector/src/main/scala/geotrellis/vector/package.scala

```diff
@@ -19,125 +19,48 @@ package geotrellis
-      with affine.MultiLineTransformationMethods
-      with dissolve.DissolveMethods[MultiLine]
-
-  implicit class MultiPolygonTransformations(val self: MultiPolygon) extends MethodExtensions[MultiPolygon]
-      with affine.MultiPolygonTransformationMethods
-      with dissolve.DissolveMethods[MultiPolygon]
-
-  implicit class GeometryCollectionTransformations(val self: GeometryCollection) extends MethodExtensions[GeometryCollection]
-      with affine.GeometryCollectionTransformationMethods
-
-  implicit def tupleOfIntToPoint(t: (Double, Double)): Point =
-    Point(t._1,t._2)
-
-  implicit def tupleOfDoubleToPoint(t: (Int, Int)): Point =
-    Point(t._1,t._2)
-
-  implicit def coordinateToPoint(c: jts.Coordinate): Point =
-    Point(c.x,c.y)
-
-  implicit def tupleSeqToMultiPoint(ts: Set[(Double, Double)]): Set[Point] =
-    ts map(t => Point(t._1, t._2))
-
-  implicit def tupleListToPointList(tl: List[(Double, Double)]): List[Point] =
-    tl map(t => Point(t._1, t._2))
-
-  implicit def tupleListToPointList(tl: Seq[(Double, Double)]): Seq[Point] =
-    tl map(t => Point(t._1, t._2))
-
-  implicit def coordinateArrayToMultiPoint(ca: Array[jts.Coordinate]): MultiPoint = {
-    val ps = (for (i <- 0 until ca.length) yield {
-      Point(ca(i).x, ca(i).y)
-    }).toSeq
-    MultiPoint(ps)
-  }
-
-  implicit def pointListToCoordinateArray(ps: List[Point]): Array[jts.Coordinate] =
-    ps map(p => new jts.Coordinate(p.x, p.y)) toArray
-
-  implicit def multiPointToSeqPoint(mp: jts.MultiPoint): Seq[Point] = {
-    val len = mp.getNumGeometries
-      (for (i <- 0 until len) yield {
-        Point(mp.getGeometryN(i).asInstanceOf[jts.Point])
-      }).toSeq
-  }
-
-  implicit def multiLineToSeqLine(ml: jts.MultiLineString): Seq[Line] = {
-    val len = ml.getNumGeometries
-    (for (i <- 0 until len) yield {
-      Line(ml.getGeometryN(i).asInstanceOf[jts.LineString])
-    }).toSeq
-  }
-
-  implicit def multiPolygonToSeqPolygon(mp: jts.MultiPolygon): Seq[Polygon] = {
-    val len = mp.getNumGeometries
-    (for (i <- 0 until len) yield {
-      Polygon(mp.getGeometryN(i).asInstanceOf[jts.Polygon])
-    }).toSeq
-  }
-
-  implicit def seqPointToMultiPoint(ps: Seq[Point]): MultiPoint = MultiPoint(ps)
-  implicit def arrayPointToMultiPoint(ps: Array[Point]): MultiPoint = MultiPoint(ps)
-
-  implicit def seqLineToMultiLine(ps: Seq[Line]): MultiLine = MultiLine(ps)
-  implicit def arrayLineToMultiLine(ps: Array[Line]): MultiLine = MultiLine(ps)
-
-  implicit def seqPolygonToMultiPolygon(ps: Seq[Polygon]): MultiPolygon = MultiPolygon(ps)
-  implicit def arrayPolygonToMultiPolygon(ps: Array[Polygon]): MultiPolygon = MultiPolygon(ps)
-
-  implicit def seqGeometryToGeometryCollection(gs: Seq[Geometry]): GeometryCollection =
-    GeometryCollection(gs)
-
   implicit class ProjectGeometry[G <: Geometry](g: G) {
     /** Upgrade Geometry to Projected[Geometry] */
     def withSRID(srid: Int) = Projected(g, srid)
   }
+
+  @typeclass trait MultiGeometry[G]
```

Comment:
This isn't used anywhere but for an import in `ProtobufGeom`. Do you think this is something important or just left-over at this point?

Reply:
Yeah, I'd agree that it's mostly vestigial.  Not entirely clear what its value was in the first place.